### ISSUE TITLE
<ui5-shellbar> support for a Logo with a customizable Background

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#1841]
**Is your feature request related to a problem? Please describe.**
The SuccessFactors Header currently has support for the logo to be placed on a "container" because the logo image does not have enough contrast with the theme background color, or maybe the image has transparency which doesn't look good on the header.

**Describe the solution you'd like**
Support an optional color to put behind the logo in a container

**Describe alternatives you've considered**
1. Make the logo as a `<slot>` so we could customize it completely
2. Stop supporting the logo container and force customer to use image that always works.

The below image shows an example:
![image](https://user-images.githubusercontent.com/730310/85085437-b34e1c80-b18c-11ea-975c-a92cb3bef338.png)

This is not a good example - I just did it myself with some sample image and purple container background color.

Basically all we need is to have that tab shape box around the logo and we just want to customize the color behind the image.

**Comment by ilhan007** [comment]
We replace the existing property "logo" with a slot with the same name to enable users set an element of their choice and freedom to customize it.

 https://github.com/SAP/ui5-webcomponents/issues/1841

The new way to set a logo (after we release) would be:
```html
<ui5-shellbar>
	<img slot="logo" src="../../../assets/images/sap-logo-svg.svg"/>
</ui5-shellbar>
```

**Comment by codefactor** [comment]
Is there any way to make this backwards compatible?

**Comment by ilhan007** [comment]
We can not have property and slot with the same name "logo", so to make it compatible we would need to anme the slot differently and keep the "logo" property. We will discuss this on monday.
